app/assets/stylesheets/home.scss

```diff
@@ -16,4 +16,21 @@
   padding: 2px;
   color: red;
   display: table;
-}
\ No newline at end of file
+}
+
+div.random_card_for_check {
+  margin-top: 50px;
+  padding-bottom: 20px;
```

Comment:
Color literals like `#f0ad4e` should only be used in variable declarations; they should be referred to via variable everywhere else.